websocket/backend_server/server.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/random;
+import ballerina/websocket;
+
+type Location record {|
+    float latitude;
+    float longitude;
+|};
+
+// Example ws://localhost:9091/logistics/vehicles/V-20.
+service /logistics on new websocket:Listener(9091) {
+    resource function get vehicles/[string vehicleId]() returns websocket:Service {
+        return new OrderService(vehicleId);
+    }
+}
+
+distinct service class OrderService {
```

Comment:
Do you think `OrderService ` is a suitable name for this service?

Reply:
Will update the name as `LocationService`.